### DISCUSSION TITLE
[Question]:

### DISCUSSION THREAD

**Discussion by rajabhupati** [danny-avila/LibreChat#2009] [Q&A]
### What is your question?

Couple of questions , do we have an api level exposure for each model so that this can be used by external applications 

Also, can all the prompts and session can be stored in some database to monitor what users are sending to model api? I believe that’s what mongo db for? Where can I find more information?

### More Details

We are looking for api for every model so that it can be consumed by external applications and process 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment]
> do we have an api level exposure for each model so that this can be used by external applications
- No, LibreChat goal is to be the frontend solution for various APIs

> Also, can all the prompts and session can be stored in some database to monitor what users are sending to model api? I believe that’s what mongo db for? Where can I find more information?
- Exact, everything is stored in mongodb, if you are using docker you can use this guide for a way to monitor and manage the db content: https://docs.librechat.ai/features/manage_your_database.html

**Reply by danny-avila** [reply]
> Couple of questions , do we have an api level exposure for each model so that this can be used by external applications

To add to this, I'm not completely against an admin deliberately exposing the API for external apps in some secure manner, but it's not a priority, nor planned or intended by default. 

Given the current auth schemas, it would be cumbersome to use as an API anyway.
